modules/oe_content_sub_entity/src/Entity/SubEntityBase.php

```diff
@@ -127,4 +127,12 @@ public function getParentEntity(): ?ContentEntityInterface {
     return $parent;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function getBundleName(): ?string {
```

Comment:
Remove this as we will use hook alter to add the bundle name to the IEF widget.

Reply:
Removed.